walk-in-interview/src/test/java/com/google/job/data/JobsDatabaseTest.java

```diff
@@ -245,6 +246,182 @@ public void fetchJob_normalInput_success() throws ExecutionException, Interrupte
+                .build();
+
+        Job job2 = Job.newBuilder()
+                .setJobStatus(jobStatus)
+                .setJobTitle(jobName)
+                .setJobDescription(jobDescription)
+                .setRequirements(requirements)
+                .setPostExpiry(postExpiry)
+                .setJobDuration(jobDuration)
+                .setLocation(location2)
+                .setJobPay(jobPayment2)
+                .build();
+
+        Job job3 = Job.newBuilder()
+                .setJobStatus(jobStatus)
+                .setJobTitle(jobName)
+                .setJobDescription(jobDescription)
+                .setRequirements(requirements)
+                .setPostExpiry(postExpiry)
+                .setJobDuration(jobDuration)
+                .setLocation(location3)
+                .setJobPay(jobPayment3)
+                .build();
+
+        Job job4 = Job.newBuilder()
+                .setJobStatus(jobStatus)
+                .setJobTitle(jobName)
+                .setJobDescription(jobDescription)
+                .setRequirements(requirements)
+                .setPostExpiry(postExpiry)
+                .setJobDuration(jobDuration)
+                .setLocation(location4)
+                .setJobPay(jobPayment4)
+                .build();
+
+        Job job5 = Job.newBuilder()
+                .setJobStatus(jobStatus)
+                .setJobTitle(jobName)
+                .setJobDescription(jobDescription)
+                .setRequirements(requirements)
+                .setPostExpiry(postExpiry)
+                .setJobDuration(jobDuration)
+                .setLocation(location5)
+                .setJobPay(jobPayment5)
+                .build();
+
+        Job job6 = Job.newBuilder()
+                .setJobStatus(jobStatusExpired)
+                .setJobTitle(jobName)
+                .setJobDescription(jobDescription)
+                .setRequirements(requirements)
+                .setPostExpiry(postExpiry)
+                .setJobDuration(jobDuration)
+                .setLocation(location6)
+                .setJobPay(jobPayment6)
+                .build();
+    
+        // the jobs will be added in a random order
+        firestore.collection(TEST_JOB_COLLECTION).add(job5).get();
+        firestore.collection(TEST_JOB_COLLECTION).add(job1).get();
+        firestore.collection(TEST_JOB_COLLECTION).add(job3).get();
+        firestore.collection(TEST_JOB_COLLECTION).add(job4).get();
+        firestore.collection(TEST_JOB_COLLECTION).add(job2).get();
+        firestore.collection(TEST_JOB_COLLECTION).add(job6).get();
+
+        JobPage expectedJobPage = new JobPage(/* jobList= */ Arrays.asList(job1, job2, job3),
+            /* totalCount= */ 3, Range.between(1, 3));
+
+        // sorting is already defaulted to SALARY and ordering is defaulted to DESCENDING
+        // maxLimit is defaulted to Integer.MAX_VALUE
+        JobQuery jobQuery = new JobQuery().setMinLimit(104001).setRegion(SingaporeRegion.CENTRAL);
```

Comment:
104001 <-- i recommend you define a variable and use that variable in the query and in the job definition or something like that. It can be the exact job salary or it can be something like `int minLimit = job1Salary - 1;`. However you prefer, the idea being that you want to link that value to the data that you arranged above.